app/styles/app.scss

```diff
@@ -20,16 +20,30 @@
   list-style-type: none;
 }
 
-
-
 .reviews-brand {
-  background: url('img/reviews-logo.png') top center no-repeat;
-  height: 70px;
-  background-size: contain;
+  background: url('img/reviews-logo-beta.png') top center no-repeat;
```

Comment:
@chrisseto, that is what I suggested for the first release (it was approved). The first release will most likely be private beta to only the current preprint providers. I imagine beta would come off once OSF admin aren't required to give admin the permissions to setup moderation.

I haven't heard anything against labelling it as beta, have you?